model/factories.go

```diff
@@ -125,3 +125,113 @@ func WaterTemperatureSensor() *SensorV2 {
 		Format:   keyRound1,
 	}.SensorV2()
 }
+
+// Variable factories //
+
+// NewAlarmNetworkVar returns a new AlarmNetwork var with the
+// passed number of failures as its value.
+func NewAlarmNetworkVar(numberOfFailures string) *Variable {
+	return &Variable{
+		Name:  "AlarmNetwork",
+		Value: numberOfFailures,
+	}
+}
+
+// NewAlarmPeriodVar returns a new AlarmPeriod var with the
+// passed duration (in seconds) as its value.
+func NewAlarmPeriodVar(duration string) *Variable {
+	return &Variable{
+		Name:  "AlarmPeriod",
+		Value: duration,
+	}
+}
+
+// NewAlarmRecoveryVoltageVar returns a new AlarmRecoveryVoltage var with the
+// passed threshold as its value.
+func NewAlarmRecoveryVoltageVar(threshold string) *Variable {
+	return &Variable{
+		Name:  "AlarmRecoveryVoltage",
+		Value: threshold,
+	}
+}
+
+// NewAlarmVoltageVar returns a new AlarmVoltage var with the
+// passed threshold as its value.
+func NewAlarmVoltageVar(threshold string) *Variable {
+	return &Variable{
+		Name:  "AlarmVoltage",
+		Value: threshold,
+	}
+}
+
+// NewAutoRestartVar returns a new AutoRestart var with the
+// passed duration (in seconds) as its value.
+func NewAutoRestartVar(duration string) *Variable {
+	return &Variable{
+		Name:  "AutoRestart",
+		Value: duration,
+	}
+}
+
+// NewPower1Var returns a new Power1 var with the
+// power initialised to OFF.
+func NewPower1Var() *Variable {
```

Comment:
Similarly, this should take a `bool` which is converted to either "true" or "false".